apps/condo/domains/common/utils/iframe.utils.ts

```diff
@@ -131,6 +147,12 @@ const SystemMessageSchema = {
     type: 'object',
     properties: {
         type: { enum: AvailableMessageTypes },
+        taskId: { type: 'string' },
+        taskTitle: { type: 'string' },
+        taskDescription: { type: 'string' },
+        taskProgress: { type: 'number' },
+        taskStatus: { enum: ['processing', 'completed', 'error'] },
+        taskOperation: { enum: ['create', 'update'] },
```

Comment:
We have a mess of all properties here, belonging to different parts of the system. The prefix `task…` is a workaround of this complexity.
My original idea was to incapsulate all implementation related to tasks as maximally, as possible to make it clear, where tasks handling implementation is.
That's why I've created a component `MiniAppTaksPostMessageAdapter`
Could you follow this approach?
Is it possible to move all implementation of handling tasks via postMessage into `MiniAppTaksPostMessageAdapter`, mount it into `GlobalAppsComponent` and write a little glue-code?

Reply:
All postMessage handling should be in single place (actually 2 places, one for global apps and one for page-apps)